docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/FastForest.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic.Trainers.Regression
+{
+    public static class FastForest
+    {
+        // This example requires installation of additional NuGet package
+        // <a href="https://www.nuget.org/packages/Microsoft.ML.FastTree/">Microsoft.ML.FastTree</a>.
+        public static void Example()
+        {
+            // Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
+            // as a catalog of available operations and as the source of randomness.
+            // Setting the seed to a fixed number in this example to make outputs deterministic.
+            var mlContext = new MLContext(seed: 0);
+
+            // Create a list of training examples.
+            var examples = GenerateRandomDataPoints(1000);
+
+            // Convert the examples list to an IDataView object, which is consumable by ML.NET API.
+            var trainingData = mlContext.Data.LoadFromEnumerable(examples);
+
+            // Define the trainer.
+            var pipeline = mlContext.Regression.Trainers.FastForest();
```

Comment:
Maybe user wants to learn some little configuration. Could you somehow make it non-default?

Reply:
I had thought about this, and decided to keep it as defaults for the following reasons:

1) ML.NET has done some work with tuning the defaults so that users can have a good starting point. If we add parameters here, users would think that they have to worry about coming up with those parameters for their own scenario. And that's a negative. It's much easier for users to think that they don't need any parameter, until they actually know what they're doing.

2) Any .NET developer can easily find out what parameters are available. The real value is not to show them the parameters, but to explain why they might need to change say number of trees or leaves. We're already doing that in the parameter description, and it would be too much to go over those details in the sample body.


---
In reply to: [266665195](https://github.com/dotnet/machinelearning/pull/2999#discussion_r266665195) [](ancestors = 266665195)

Reply:
I think there is another example in `FastForestWithOptions.cs` that shows the configuration.

---
In reply to: [267017735](https://github.com/dotnet/machinelearning/pull/2999#discussion_r267017735) [](ancestors = 267017735,266665195)